sdk/rust/crates/dagger-sdk/src/core/downloader.rs

```diff
@@ -149,7 +149,7 @@ impl Downloader {
         let actual_hash = self.extract_cli_archive(&mut bytes).await?;
 
         if expected_checksum != actual_hash {
-            eyre::bail!("downloaded CLI binary checksum doesn't match checksum from checksums.txt")
+            //eyre::bail!("downloaded CLI binary checksum: {actual_hash} doesn't match checksum from checksums.txt: {expected_checksum}")
```

Comment:
Should this be uncommented now that https://github.com/dagger/dagger/issues/5944 is fixed?

Reply:
Ah yeah, I was testing some stuff until: https://github.com/dagger/dagger/issues/5944 was fixed =D

Reply:
All good! Just restored this.